public/scripts/extensions/connection-manager/index.js

```diff
@@ -357,6 +390,9 @@ async function renderDetailsContent(detailsContent) {
     const profile = extension_settings.connectionManager.profiles.find(p => p.id === selectedProfile);
     if (profile) {
         const profileForDisplay = makeFancyProfile(profile);
+        if (Array.isArray(profile.exclude) && profile.exclude.length > 0) {
+            profileForDisplay['Omitted Settings'] = profile.exclude.map(e => FANCY_NAMES[e]).join(', ');
```

Comment:
Using this looks a bit weird. As on first glance it looks like another line of the profile with it's value.

I mean, that's the easiest solution, so keep it if you want. But I think having it on a line below, not as part of the list, would make more sense.

![image](https://github.com/user-attachments/assets/f1f05cd8-32bb-4ce5-9659-d860a62591b4)